src/config.js

```diff
@@ -0,0 +1,41 @@
+const config = {
+  BODY: document.getElementsByTagName('body')[0],
+
+  REGEX: {
+    IDEVICE          : /iPad|iPhone|iPod/,
+    ANDROID          : /android/i,
+    LEADING_SLASH    : /(^\/?)/,
+    COOKIE           : /(?:^|; )'+a+'=([^;]*)/,
+    QUERY_PARAMS     : /[?&]+([^=&]+)=([^&]*)/gi,
+    HTTP_PROTOCOL    : /^https?:\/\//,
+    MAILTO_HREF      : /^mailto:/,
+    TEL_HREF         : /^TEL:/,
+    SPACES           : /\s+/g,
+    NON_WORD         : /[^\w\-]+/g,
+    MULTIPLE_HYPHENS : /\-\-+/g,
+    LEADING_HYPHENS  : /^-+/,
+    TRAILING_HYPHENS : /-+$/
+  },
+
+  DEFAULTS: {
+    debug: false,
+    redirectDelay : 100,
+    files: '.pdf',
+    networks: 'facebook\.com|twitter\.com|instagram\.com|linkedin\.com|pinterest\.com|tumblr\.com|plus\.google\.com'
```

Comment:
It seeeeeems like this should be in an array that gets assembled into a string and then a `Regex` object. But then, not sure who will be editing this list or why. Something like this, but not as ugly:

```
new RegExp([
  'test.com',
  'www.facebook.com'
].map((s) => s.replace(/\./g, '\\.')).join('|'))
```